### ISSUE TITLE
[BUG] withDlq without specifiying name, raise an error

### ISSUE THREAD

**Issue by nmfzone** [mateusjunges/laravel-kafka#94]
If we're configuring DLQ with below syntax (from the docs):

```
$consumer = \Junges\Kafka\Facades\Kafka::createConsumer()->withDlq();
```

It would raise an error 

```
In ConsumerBuilder.php line 221:
                         
  Undefined array key 0 
```

Package version 1.7.3

**Comment by mateusjunges** [comment]
Hi @nmfzone the docs are wrong actually. You can't specify a DLQ without subscribing to any topics. I will fix it asap. Thanks for reporting.

**Comment by mateusjunges** [comment]
@nmfzone fixed in v1.7.4. I've also updated the docs: https://junges.dev/documentation/laravel-kafka/v1.7/consuming-messages/5-configuring-consumer-options

Thank you!

**Comment by nmfzone** [comment]
Awesome, thank you. That was fast :)
